src/main/java/org/jabref/logic/exporter/CffExporter.java

```diff
@@ -0,0 +1,227 @@
+                    Map<String, String> authorMapPreferred = new LinkedHashMap<>();
+                    if (author.getFamilyName().isPresent()) {
+                        authorMap.put("family-names", author.getFamilyName().get());
+                        authorMapPreferred.put("family-names", author.getFamilyName().get());
+                    }
+                    if (author.getGivenName().isPresent()) {
+                        authorMap.put("given-names", author.getGivenName().get());
+                        authorMapPreferred.put("given-names", author.getGivenName().get());
+                    }
+                    if (author.getNamePrefix().isPresent()) {
+                        authorMap.put("name-particle", author.getNamePrefix().get());
+                        authorMapPreferred.put("name-particle", author.getNamePrefix().get());
+                    }
+                    if (author.getNameSuffix().isPresent()) {
+                        authorMap.put("name-suffix", author.getNameSuffix().get());
+                        authorMapPreferred.put("name-suffix", author.getNameSuffix().get());
+                    }
+                    authorsList.add(authorMap);
+                    authorsListPreferred.add(authorMapPreferred);
+                });
+                data.put("authors", authorsList.isEmpty() ? "No author specified." : authorsList);
+                entryMap.remove(StandardField.AUTHOR);
+
+                // Type
+                Map<EntryType, String> typeMap = getTypeMappings();
+                EntryType entryType = entry.getType();
+                switch (entryType) {
+                    case StandardEntryType.Software, StandardEntryType.Dataset ->
+                            data.put("type", entryType.getName());
+                    default -> {
+                        if (typeMap.containsKey(entryType)) {
+                            data.put("preferred-citation", preferredData);
+                            data = preferredData;
+                            data.put("type", typeMap.get(entryType));
+                            data.put("authors", authorsListPreferred.isEmpty() ? "No author specified." : authorsListPreferred);
+                            data.put("title", title);
+                        }
+                    }
+                }
+
+                // Keywords
+                String keywords = entryMap.getOrDefault(StandardField.KEYWORDS, null);
+                if (keywords != null) {
+                    data.put("keywords", keywords.split(",\\s*"));
+                }
+                entryMap.remove(StandardField.KEYWORDS);
+
+                // Date
+                String date = entryMap.getOrDefault(StandardField.DATE, null);
+                if (date != null) {
+                    String formatString;
+                    try {
+                        LocalDate localDate = LocalDate.parse(date, DateTimeFormatter.ISO_LOCAL_DATE);
```

Comment:
I think, JabRef has an own date parsing part. Maybe, you find it. Either use it or add a comment, why you don't use it.